lcls_tools/superconducting/scLinac.py

```diff
@@ -21,21 +21,21 @@ def __init__(self, cavity):
         
         if self.cavity.cryomodule.is_harmonic_linearizer:
             cavity_num = utils.HL_SSA_MAP[self.cavity.number]
-            hl_prefix = "ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:SSA:".format(LINAC=self.cavity.linac.name,
+            self.hl_prefix = "ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:SSA:".format(LINAC=self.cavity.linac.name,
                                                                          CRYOMODULE=self.cavity.cryomodule.name,
                                                                          CAVITY=cavity_num)
             self.fwd_power_lower_limit = 500
             
-            self.ps_volt_setpoint1_pv: str = hl_prefix + "PSVoltSetpt1"
+            self.ps_volt_setpoint1_pv: str = self.hl_prefix + "PSVoltSetpt1"
             self._ps_volt_setpoint1_pv_obj: PV = None
             
-            self.ps_volt_setpoint2_pv: str = hl_prefix + "PSVoltSetpt2"
+            self.ps_volt_setpoint2_pv: str = self.hl_prefix + "PSVoltSetpt2"
             self._ps_volt_setpoint2_pv_obj: PV = None
             
-            self.status_pv: str = (hl_prefix + "StatusMsg")
-            self.turn_on_pv: str = (hl_prefix + "PowerOn")
-            self.turn_off_pv: str = (hl_prefix + "PowerOff")
-            self.reset_pv: str = hl_prefix + "FaultReset"
+            self.status_pv: str = (self.hl_prefix + "StatusMsg")
+            self.turn_on_pv: str = (self.hl_prefix + "PowerOn")
+            self.turn_off_pv: str = (self.hl_prefix + "PowerOff")
+            self.reset_pv: str = self.hl_prefix + "FaultReset"
```

Comment:
Do "NRP_PRMT" and "FaultSummary.SEVR" need to be added to this section too or nah?

Reply:
I don't think so unless we think other screens are gonna use them. I added them to the utils list though